src/k8s_client/k8s_client.go

```diff
@@ -615,3 +617,14 @@ func (c *k8sClient) PatchNodeLabels(nodeName string, nodeLabels string) error {
 	_, err := c.client.CoreV1().Nodes().Patch(context.Background(), nodeName, types.MergePatchType, data, metav1.PatchOptions{})
 	return err
 }
+
+func (c *k8sClient) IsClusterCapabilityEnabled(capability configv1.ClusterVersionCapability) (result bool, err error) {
+	version, err := c.GetClusterVersion()
+	if err != nil {
+		return
+	}
+	isExplicitlyEnabled := funk.Contains(version.Status.Capabilities.EnabledCapabilities, capability)
+	isKnown := funk.Contains(version.Status.Capabilities.KnownCapabilities, capability)
+	result = isExplicitlyEnabled || !isKnown
+	return
+}
```

Comment:
nit: Named return values seem unnecessary in this simple function

```suggestion
func (c *k8sClient) IsClusterCapabilityEnabled(capability configv1.ClusterVersionCapability) (bool, error) {
	version, err := c.GetClusterVersion()
	if err != nil {
		return false, err
	}
	isExplicitlyEnabled := funk.Contains(version.Status.Capabilities.EnabledCapabilities, capability)
	isKnown := funk.Contains(version.Status.Capabilities.KnownCapabilities, capability)
 	return isExplicitlyEnabled || !isKnown
}
```